sympy/ntheory/residue_ntheory.py

```diff
@@ -829,15 +829,24 @@ def quadratic_residues(p):
 
 
 def legendre_symbol(a, p):
-    """
-    Returns
-    =======
+    r"""
+    Returns the Legendre symbol ``(a / p)``.
```

Comment:
Should be single backquotes for math (doesn't make sense as code, since it's not literally `a` divided by `p`).